platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/AdminConsoleService.java

```diff
@@ -416,6 +418,15 @@ public void updateForLocation(
     }
   }
 
+  private Object sanitizeUIConfiguration(String pid, String configEntryKey, Object configEntryValue) {
+    if (UI_CONFIG_PID.equals(pid)) {
+      if ("color".equals(configEntryKey) || "background".equals(configEntryKey)) {
```

Comment:
I was able to bypass this check by intercepting the outgoing request and changing the "color" key to "Color" (or any other case) and could reproduce the attack. Background is similar. It looks like whatever is looking up the keys to set the style value doesn't care about case.

Reply:
It also threw an error "Key [Color] already present in different case" when I tried to change it through the UI (which uses lowercase) aftewards so it looks like this issue can only occur if it's never been set; afterwhich, it requires that specific case. So we might want to change any incoming keys to lower-case before persisting them to prevent that error.